package.json

```diff
@@ -154,6 +154,6 @@
     ]
   },
   "dependencies": {
-    "auth0-chrome": "github:alpheios-project/auth0-chrome#feature-logout"
+    "auth0-chrome": "github:alpheios-project/auth0-chrome#ff-support"
```

Comment:
should this be master now? are you merging the logout change to both ff-support and master or merging both to master?

Reply:
Oh, that's correct, will fix that!